documentation/api/OpenApi.yaml

```diff
@@ -0,0 +1,997 @@
+openapi: 3.0.3
```

Comment:
We have the file in the frontend and here. How do we want to handle it?

Reply:
Yes true. I think it could be hard to find in to UI which is why I put there as well. Could we automatically copy it from here to the UI? Would that make sense?

Reply:
I'll deal with this issue later.